course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -286,22 +293,17 @@ describe('Unit testing PeopleController', function() {
 
         describe('closeAlert', function() {
             it('should work when an alert is present', function() {
-                scope.testAlerts = [{}];  // contents don't matter
-                scope.closeAlert('testAlerts', 0);
-                expect(scope.testAlerts).toEqual([]);
+                scope.testAlerts = 'This is a test';  // contents don't matter
```

Comment:
in this case, since the closeAlert method was changed, we want to make sure it clears out the contents of the specified alert. In this case it's the string 'This is a test'